test/Models/Catalog/Ows/WebProcessingServiceCatalogFunctionSpec.ts

```diff
@@ -148,7 +148,7 @@ describe("WebProcessingServiceCatalogFunction", function () {
       let dispose: any;
       job = (await wps.submitJob()) as WebProcessingServiceCatalogFunctionJob;
 
-      await new Promise<void>((resolve, reject) => {
+      await new Promise<void>((resolve, _reject) => {
```

Comment:
The change in YDYRCatalogFunctionSpec above just removes the parameter. If this isn't somehow different from that, I think it's better to either choose to always keep the unused reject with an underscore or always remove it.

Reply:
I'll remove the unused rejects since it's not communicating anything extra